client/app/previous-pres/previousPres.controller.js

```diff
@@ -52,3 +62,48 @@ angular.module('spkr.previous-pres', ['ngRoute'])
     })
 
   });
+
```

Comment:
This is probably because it was copy/pasted from another site, but the indentation is four lines, not two lines. Please fix this inconsistency. (There are tools/settings in sublime for doing this.)

Also, please include a newline at the end of the file (and all files).